ambuda/__init__.py

```diff
@@ -119,4 +119,9 @@ def create_app(config_env: str):
         }
     )
 
+    @app.after_request
+    def add_security_headers(resp):
+        resp.headers["Content-Security-Policy"] = "default-src 'self'; script-src 'self' https://cdn.jsdelivr.net https://www.google.com https://www.gstatic.com https://plausible.io; frame-src https://www.google.com;"
```

Comment:
what do we load from google and gstatic?

Reply:
reCAPTCHA